test/naming_tests.jl

```diff
@@ -26,11 +26,13 @@ n = (x = x, y = y)
 
 z = "lala"
 d2 = Dict(:x => x, :y => y, :z => z)
-n2 = (x = x, y = y, z= z)
+n2 = (x = x, y = y, z = z)
 
 @test d2 == @dict x y z
 @test n2 == @ntuple x y z
 
+@test savename(n2; ignores=(:y,)) == "x=3_z=lala"
```

Comment:
A case where both `ingores` and `accesses` are specified must be tested, to ensure that the set difference indeed works as expected.

Reply:
Added.